fmriprep/cli/sample_openfmri_tasks_list.py

```diff
@@ -56,30 +64,49 @@ def main():
     if opts.anat_only:
         cmdline += ' --anat-only'
 
+    if opts.nthreads:
+        cmdline += '--nthreads %d' % opts.nthreads
+
+    if opts.omp_nthreads:
+        cmdline += '--omp-nthreads %d' % opts.omp_nthreads
+
+    if opts.mem_gb:
+        cmdline += '--mem_mb %d' % (opts.mem_gb * 1000)
+
+    if opts.tasks_filter:
+        cmdline += '-t %s' % ' '.join(opts.tasks_filter)
+
     fmriprep_cmd = 'fmriprep'
     if opts.cmd_call is None:
         singularity_dir = os.getenv('SINGULARITY_BIN')
         singularity_img = sorted(
-            glob.glob(os.path.join(singularity_dir, 'poldracklab_fmriprep_1*')))
+            glob.glob(os.path.join(singularity_dir, 'poldracklab_fmriprep_*')))
         if singularity_img:
             fmriprep_cmd = 'singularity run %s' % singularity_img[-1]
 
-
     task_cmds = []
-    for dset, sublist in sampledict.items():
-        os.mkdir(dset)
 
+    # Try to make this Python 2 compatible
+    try:
+        os.makedirs(opts.output_dir)
+    except OSError, e:
+        if e.errno != os.errno.EEXIST:
+            raise
+
+    for dset, sublist in sampledict.items():
```

Comment:
Do we not need to mkdir `output_dir/dataset_dir` anymore?

Reply:
We don't :)